src/content/en/fundamentals/performance/audit/index.md

```diff
@@ -81,20 +85,24 @@ If nothing else, simply use [Lighthouse](/web/tools/lighthouse/) to check for:
 
   * HTTPS: [every site should deliver all assets over HTTPS](/web/fundamentals/security/encrypt-in-transit/why-https).
   * Server settings: your web server or CDN should [use compression correctly](/web/fundamentals/performance/optimizing-content-efficiency/optimize-encoding-and-transfer), [use HTTP/2](/web/fundamentals/performance/http2/), and [include appropriate headers](/web/fundamentals/performance/optimizing-content-efficiency/http-caching) to enable your browser to cache resources.
-  * Script elements that can be moved to the bottom of the page and/or given an [async or defer](http://peter.sh/experiments/asynchronous-and-deferred-javascript-execution-explained/) attribute.
+  * Script elements that can be moved to the bottom of the page and/or given 
+an [async or defer](http://peter.sh/experiments/asynchronous-and-deferred-javascript-execution-explained/){: .external} attribute.
   * JavaScript and libraries that can be removed.
-  * [Unused CSS](https://umaar.com/dev-tips/121-css-coverage/) and [unused JavaScript](/web/updates/2017/04/devtools-release-notes).
+  * [Unused CSS](https://umaar.com/dev-tips/121-css-coverage/){: .external} 
+and [unused JavaScript](/web/updates/2017/04/devtools-release-notes).
```

Comment:
You should indent this line so it's inline with the previous

Reply:
Done.